src/component/welcome-logout/WelcomeLogout.jsx

```diff
@@ -4,10 +4,11 @@ import styles from './WelcomeLogout.module.scss';
 
 const WelcomeLogout = ({ firstName, logout }) => {
     return (
-        <div className={styles.logoutContainer}>
-            <p className={styles.welcomeMessage}>Welcome, {firstName}!</p>
-            <button className={styles.logoutButton} onClick={logout}>Logout</button>
-        </div>
+        <>
+            <li className={styles.a}>Welcome, {firstName}!</li>
```

Comment:
Considering we already have welcome on the `org-dashboard` already, can we  remove the `Welcome` here? OR we can add the first name next to the welcome found on the `org-dashboard` and then have this to just have the logout button.